install.sh

```diff
@@ -198,6 +198,33 @@ if [ "$ZOOKEEPER_SNAPSHOT_FOLDER_EXISTS" -eq "1" ]; then
   fi
 fi
 
+# [begin] Snuba/Clickhouse transactions table rebuild
+clickhouse_query () { $dcr clickhouse clickhouse-client --host clickhouse -q "$1"; }
+$dc up -d clickhouse
+set +e
+CLICKHOUSE_CLIENT_MAX_RETRY=5
+# Wait until clickhouse server is up
+until clickhouse_query 'SELECT 1' > /dev/null; do
+  ((CLICKHOUSE_CLIENT_MAX_RETRY--))
+  [[ CLICKHOUSE_CLIENT_MAX_RETRY -eq 0 ]] && echo "Clickhouse server failed to come up in 5 tries." && exit 1;
+   echo "Trying again. Remaining tries #$CLICKHOUSE_CLIENT_MAX_RETRY"
+  sleep 0.5;
+  SNUBA_TRANSACTIONS_NEED_UPGRADE=$(clickhouse_query "SHOW CREATE TABLE transactions_local" | grep -v 'SAMPLE BY cityHash64(span_id)')
+done
+set -e
+
+if [ ! -z "$SNUBA_TRANSACTIONS_NEED_UPGRADE" ]; then
```

Comment:
not very familiar with bash scripting, but is this the simplest for for this condition? it took me a while to figure out that if you did not to update the variable would be empty, that `-z` returns true if it is empty and that we negate it.

Reply:
Okay, I think this new version is clearer and also is the correct logic.